bootstrap.bat

```diff
@@ -1,7 +1,9 @@
+@REM This is mostly for windows 7. Recommend to run bootstrap.ps1 in Windows 10 instead.
 @REM Author: Walter Chen
 @REM Version: 0.1.1
 @REM 0.1.0 - Successfully running in virtual machines, appveyor, and real machines.
 @REM 0.1.1 - Adding instruction and changed the command for detecting admin privileges.
+@REM 0.1.2 - Backing up the PATH variable for you in path.txt before running bootstrap
```

Comment:
how about using a better file name for this?  ex: bootstrap_backup_env_path.txt ?